### PR TITLE
Add docs

### LINKED ISSUE #53: Documentation
There are already some docs in the upstream wiki, but some of the local customizations
should have documentation too:

* [ ] HV Reactor digilines
* [ ] HV Digilines cable
* [ ] (other stuff i forgot)

A simple markdown documentation should suffice

### PR DESCRIPTION
**Pull request by Athozus**
Based on @BuckarooBanzay post :

* [x] HV Digilines cable
* [x] Polyfuse
* [x] Overload (cheat mitigation)
* [x] Chest sort modes and inventory move buttons
* [x] Chest `digilines` functionality
* [x] Multimeter

I didn't added the non-checked documentations because I don't know what you mean with, so I also made this PR as a draft.

EDIT : I've added those in the commit de3e6b9

Helps with #53

### CODE REVIEW

File: technic/doc/chests.md
Comment (S-S-X) [inline technic/doc/chests.md:56]:
```suggestion
Chests can be sorted with one button with the following sort types : natural sort, by item, by quantity, by type or by wear. The chests can also be sorted automatically when its content changes.
```

File: technic/doc/digilines.md
Comment (S-S-X) [inline technic/doc/digilines.md:9]: Maybe just drop "new", reactor does not have to be bottom, also few other suggestions.
```suggestion
Digicables are a type of cable that carry both electric power and digiline messages. Like both original types of cable (electric cable and digiline), you just have to connect cables to each other in order to transmit power/information. They are needed to access nuclear reactor digiline information. The digicables are available in HV, MV and LV.
```

I think important thing to add would be crafting because it needs alloying and therefore might not be that obvious especially if crafting guide isn't able to display alloying recipes (or if playing without guide).

Comment (S-S-X) [reply technic/doc/digilines.md:9]: But then also, recipes maybe do not belong here in this file. Not sure if those are already somewhere but probably not...

File: technic/doc/tools.md
Comment (S-S-X) [inline technic/doc/tools.md:49]: For multimeter it would be very good to actually document values it displays, few are obvious but many are not if player isn't very familiar with technical stuff under the hood.
Maybe table with descriptions for each value displayed by multimeter.

File: technic/doc/digilines.md
Comment (OgelGames) [reply technic/doc/digilines.md:9]: IMO it's better not to document recipes, because they can often be different. Servers change recipes quite often.

It should state the difference in connecting rules though, digicables connect differently than standard digilines.

File: technic/doc/chests.md
Comment (OgelGames) [reply technic/doc/chests.md:56]: There are actually three buttons, one to change the sort method, one to sort, and one to toggle auto-sorting (except for iron chests, which lack auto-sorting).

Really the chest docs need a lot more changed, particularly when it comes to the mithril chests (which is what the "Chest digilines functionality" task is about).

**Comment by Athozus** [comment]
I followed your suggestions, but I wrote basic meanings for "Timeout" and "Skip" because I'm not sure what it really means.

File: technic/doc/digilines.md
Comment (S-S-X) [reply technic/doc/digilines.md]: I've not actually ever seen a single change change in any of special recipes (that are not crafting which in turn works in all guides). Probably those too have changed outside the mods but definitely not often.

File: technic/doc/tools.md
Comment (S-S-X) [inline technic/doc/tools.md:53]: This actually isn't pointing to switching station but 1 node below it which is cable.
While it is very often directly below switching station this is not always the case, there does not have to be switching station above it.

Actually this information is technical detail that isn't too simple when you dive into it.
Reason it is included is that I first created multimeter to aid debugging during development but just tried to make it also usable by regular players.

Comment (S-S-X) [reply technic/doc/tools.md:53]: Not sure what to change in description though... Network ID is derived from this coordinate (for both actual network implementation and for ID that shows up in multimeter).
Coordinate is also used with administrative commands, same location those commands will show.

File: technic/doc/digilines.md
Comment (OgelGames) [reply technic/doc/digilines.md]: Yeah, probably a bad assumption, but I have seen recipes changed. I do remember some technic recipes changed on Tunnelers' Abyss for example.

Comment (S-S-X) [reply technic/doc/digilines.md]: > technic recipes changed on Tunnelers' Abyss for example.

They're not using either Technic plus nor original Technic mod, they're using their own fork which has their own documentation. Same goes with all of the servers I've seen changing their recipes.

File: technic/doc/chests.md
Comment (S-S-X) [reply technic/doc/chests.md]: Additionally chests documentation should probably be moved under chests mod instead of keeping it in technic mod.
I think it would probably be better to create another PR just for chests documentations.

File: technic/doc/tools.md
Comment (S-S-X) [inline technic/doc/tools.md:55]: Maybe
```suggestion
| Timeout          | 1800.0        | s          | Time remaining, in seconds, before the network will shut down |
```

File: technic/doc/tools.md
Comment (S-S-X) [inline technic/doc/tools.md:57]: To make in clear what affects this maybe a little addition:
```suggestion
| Skip             | 0             | cycles     | How many cycles are skipped when network lag value is high    |
```

File: technic/doc/tools.md
Comment (S-S-X) [inline technic/doc/tools.md:59]:
```suggestion
| Supply           | 6000          | EU         | How much power is supplied by generators                      |
| Demand           | 4000          | EU         | How much power is required by consumers                       |
```

File: technic/doc/tools.md
Comment (S-S-X) [inline technic/doc/tools.md:63]:
```suggestion
| Nodes            | 47            | count      | The number of nodes of the network, excluding switch stations |
```

Comment (S-S-X) [reply technic/doc/tools.md:63]: For multimeter, would probably be good to have switching station count as a separate measurement and also make waypoint functionality display waypoints for switching stations.
Not for this pull request but good update now there's actually easy straightforward way to get/store positions of all connected switching stations.

**Comment by Athozus** [comment]
Corrected the docs following your suggestions, notably multimeter properties table. Now waiting for approval.

**Comment by S-S-X** [comment]
When you think it is ready, mark as ready for review taking PR out of the draft state.
Currently however this probably would not yet be enough to close #53 so either closing keyword should be removed or rest of it implemented (at least what is listed on #53).

**Comment by OgelGames** [comment]
We might as well keep this open until it's complete, there's no rush, the documentation isn't a blocker for 2.0.0.

@Athozus do you mind if I add to this PR? I can at least do the chests and quarry docs, as those are two parts of the mod I know a lot about (because I designed them and wrote the code).

**Comment by Athozus** [comment]
> When you think it is ready, mark as ready for review taking PR out of the draft state. Also when starting reviews it would be good to rebase it to master or merge changes from master here.
> 
> Currently however this probably would not yet be enough to close #53 so either closing keyword should be removed or rest of it implemented (at least what is listed on #53).

Was too fast, forgot that there was still docs for polyfuse, overload and chests digilines. I don't know what means all those, the behaviour, ... so I need someone to explain me a bit, or write the doc.

**Comment by Athozus** [comment]
> We might as well keep this as a draft until it's complete, there's no rush, the documentation isn't a blocker for 2.0.0.
> 
> @Athozus do you mind if I add to this PR? I can at least do the chests and quarry docs, as those are two parts of the mod I know well (because I designed and wrote the code of the new versions).

add what you want to the PR, np (i may squash my commits later)

**Comment by Athozus** [comment]
I just saw back this one. @S-S-X @OgelGames are you approving for merging ? Are there still changes to do ?

File: technic/doc/tools.md
Comment (S-S-X) [inline technic/doc/tools.md:53]: Here's suggestion for technically correct description, probably can't be made much simpler
```suggestion
| Ref. point       | 61, 19, 62    | Coordinate | The location of the cable below the switching station that initially triggered initialization of this network |
```

**Comment by S-S-X** [comment]
Also rebase first, simplest way to do this is "Update branch"-button and select "Update with rebase" option.

**Comment by Athozus** [comment]
Should be able to approval now.

**Review by S-S-X — APPROVED** [review]
A lot better documentation, still incomplete but in my opinion better to merge what we have here. Removed keyword for #53 as this wont still document everything, no reason to keep documentation improvements pending for months.

**Comment by OgelGames** [comment]
Yeah, it would be nice to have everything fixed in one PR, but it's been too long to keep this waiting.

**Pull request merged**
